packages/xo-server/src/api/host.mjs

```diff
@@ -19,9 +19,24 @@ const log = createLogger('xo:api:host')
 
 // ===================================================================
 
-export function setMaintenanceMode({ host, maintenance }) {
+export async function setMaintenanceMode({ host, maintenance, vmsToForceMigrate }) {
   const xapi = this.getXapi(host)
 
+  if (vmsToForceMigrate) {
+    await Promise.all(
```

Comment:
Since the number of VM is abritrary, it can be big. We have a syncEach utility that limit the concurrency of an asynchronoous task ( migrating them 10 by 10 for example). I think it could increase reliability

Reply:
Done

Reply:
How many do you suggest?

Reply:
default concurrency is 10, but 4 is also ok for me

Reply:
Done